zamba/models/config.py

```diff
@@ -355,7 +356,7 @@ class TrainConfig(ZambaBaseModel):
     """
 
     labels: Union[FilePath, pd.DataFrame]
-    data_dir: DirectoryPath = Path.cwd()
+    data_dir: Optional[DirectoryPath] = ""
```

Comment:
`Optional` means DirectoryPath can be `None` (https://pydantic-docs.helpmanual.io/usage/types/). I don't think `None` actually works otherwise you'll get `str(None) / df.filepath.path`. I think we just want this to be `DirectoryPath` (as you have in the PredictConfig)

Reply:
Yes.